app/src/main/java/com/example/recipeopedia/fragments/SavedRecipesFragment.java

```diff
@@ -0,0 +1,27 @@
+package com.example.recipeopedia.fragments;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.recipeopedia.R;
+
+
+public class SavedRecipesFragment extends Fragment {
+    public SavedRecipesFragment() {}
```

Comment:
I'd delete the empty default constructors, I don't think you will get a compiler error